libs/RobotKit/include/RobotController.h

```diff
@@ -188,6 +190,18 @@ class RobotController : public interface::RobotController
 
 	void startDisconnectionBehavior() final { stopActuators(); }
 
+	void startChooseAutonomousActivityBehavior() final
+	{
+		_lcd.turnOn();
+		_behaviorkit.chooseActivity();
```

Comment:
J'ai buté sur la compréhension de cette ligne. J'ai pensé que c'était là où se choisissait l'activité, mais non c'est juste afficher sur l'écran le menu

Reply:
Du coup ça porte encore plus à confusion puisqu'après avoir compris ça, on se demande à quel moment on choisit l'activité réellement.

Je suis confus.

Reply:
je suis d'accord avec @YannLocatelli -- le nom porte un peu à confusion.

je dirai 

```suggestion
		_behaviorkit.displayAutonomousActivitiesPrompt();
```